trade-finance-manager-ui/styles/footer.scss

```diff
@@ -1,3 +1,13 @@
-.email-wrap {
+.contact-details {
   word-break: break-word;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
```

Comment:
Thanks, I removed these margin attributes from `gef-ui` and `portal`, but left in `trade-finance-manager-ui`.

In `trade-finance-manager-ui` I changed default margin for dl, because I introduced them in multiple places at same time. Not sure if it is best decision, but maybe this is fine.

Reply:
Happy to leave this as is.